### PR TITLE
Fixes #18931: No skipped directives on Rule screen after parent ticket

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/18931

### CODE REVIEW

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/reports/ReportingService.scala [hunk 1]
```diff
@@ -59,7 +59,7 @@ trait ReportingService {
   /**
    * find rule status reports for a given rule.
    */
-  def findDirectiveRuleStatusReportsByRule(ruleId: RuleId): Box[RuleStatusReport]
+  def findDirectiveRuleStatusReportsByRule(ruleId: RuleId): Box[Map[NodeId, NodeStatusReport]]
```
Comment (ncharles) [inline webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/reports/ReportingService.scala:62]: i would love if you could change the comment

**Review by ncharles — CHANGES_REQUESTED** [review]
i am very surprised the change is so small, i ould have thought the impact of this would have been greater
can you correct the comment and self merge?

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**